org-journal.el

```diff
@@ -405,9 +413,10 @@ This runs once per date, before `org-journal-after-entry-create-hook'.")
 
 
 ;;;###autoload
-(add-hook 'calendar-today-visible-hook 'org-journal-mark-entries)
-;;;###autoload
-(add-hook 'calendar-today-invisible-hook 'org-journal-mark-entries)
+(if org-journal-enable-calendar-integration
```

Comment:
I think this breaks the autoloads, which are necessary for making org-journal work in the calendar if it hasn't been invoked yet. So the branch probably needs to happen inside `org-journal-mark-entries` instead.